src/lib/browse/layers/areas/Pollution.svelte

```diff
@@ -0,0 +1,145 @@
+<script lang="ts">
+  import { ExternalLink, HelpButton } from "lib/common";
+  import { Checkbox, Select } from "lib/govuk";
+  import { layerId } from "lib/maplibre";
+  import { RasterLayer, RasterTileSource } from "svelte-maplibre";
+  import OsOglLicense from "../OsOglLicense.svelte";
+  import SequentialLegend from "../SequentialLegend.svelte";
+
+  let show = false;
+  let opacity = 50;
+  let pollutant = "PM25_viridis";
+
+  // URLs and layers found from https://uk-air.defra.gov.uk/data/wms-services
+  // and QGIS, or http://www.extrium.co.uk/noiseviewer.html for the noise layer
+  $: info = {
+    NOx_viridis: ["22", "Data for 2022"],
+    PM25_viridis: ["21", "Data for 2022"],
```

Comment:
is this supposed to say for 2021?

Reply:
Based on the very end of https://ukair.maps.rcdo.co.uk/ukairserver/services/aq_amb_2022/PM25_viridis/MapServer/WMSServer?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetCapabilities and QGIS, no. The layer IDs are not consistent
![Screenshot from 2023-10-26 10-01-42](https://github.com/acteng/atip/assets/1664407/d3e261cb-cba6-4dfc-88af-a36884815ce1)